### ISSUE TITLE
Plugin not working for iOS

### ISSUE THREAD

**Issue by vrunoa** [avivais/phonegap-parse-plugin#70]
Hey @avivais , I install the `phonegap-parse-plugin` but it's not working on iOS. After install, I build & run the app, and in xcode console I get:

``` error
2015-07-28 18:19:37.651 Enquestapp[7925:4456806] -[__NSCFString JSONObject]: unrecognized selector sent to instance 0x114871d68
2015-07-28 18:19:38.308 Enquestapp[7925:4456806] *** WebKit discarded an uncaught exception in the webView:decidePolicyForNavigationAction:request:frame:decisionListener: delegate: <NSInvalidArgumentException> -[__NSCFString JSONObject]: unrecognized selector sent to instance 0x114871d68
```

Have any idea what it could be ? 

I'm using cordova 4.2.0

Thanks four your help.

**Comment by eddiekollar** [comment]
I'm currently working on this myself. Are you attempting to do run your app in the simulator?
It seems as though push notifications are only supported with an actual device.

**Comment by vrunoa** [comment]
Yeah, I was testing it on a Simulator, on monday i'll use a iphone 6 and let you know. I really hope this is the problem. thanks.

**Comment by vrunoa** [comment]
Got the same error testing on an iPhone 6. I have to remove the `-ObjC` flag from `Other Links Flags` on the project to work. Could the plugin not be working due to this ?

**Comment by Kunze** [comment]
@vrunoa  How can I remove this without xcode?

**Comment by vrunoa** [comment]
@Kunze mmm, I have never actually made it without XCode, but I believe you would have to change the build command-line you're using and add this flag.

**Comment by Kunze** [comment]
@vrunoa Thank

**Comment by rooswelt** [comment]
I'm facing the same problem (with -ObjC removed) and running on device (ipad air)

Have you found any solution yet?

**Comment by sharmahemu1994** [comment]
any solutions yet regarding this problem??

**Comment by sharmahemu1994** [comment]
add this ( -force_load $(BUILT_PRODUCTS_DIR)/libCordova.a ) in other links flags in xcode if you have removed -ObjC
